scripts/platforms/linkedin_apply.py

```
@@ -0,0 +1,463 @@
+        checkboxes = await page.locator(
+            "input[type='checkbox']"
+        ).all()
+
+        for checkbox in checkboxes:
+
+            try:
+
+                await checkbox.check()
+
+            except Exception:
+                pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)